### PR TITLE
EZP-29010: Move Content on the Fly common configuration to root namespace od AdminUI config

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-29010
**Requires:** https://github.com/ezsystems/ezplatform-admin-ui/pull/426

### CODE REVIEW

File: src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.js [hunk 1]
```diff
@@ -56,12 +56,9 @@ export default class ContentMetaPreviewComponent extends Component {
...
         }
 
-        const version = response.View.Result.searchHits.searchHit[0].value.Content.CurrentVersion.Version
+        const version = response.View.Result.searchHits.searchHit[0].value.Content.CurrentVersion.Version;
         const imageField = version.Fields.field.find(field => field.fieldTypeIdentifier === 'ezimage');
-        const translations = version.VersionInfo.VersionTranslationInfo.Language.map(langauge => {
-            // @TODO: pass it in props when COTF is cleaned
-            return window.eZ.adminUiConfig.languages.map[langauge.languageCode].name;
-        });
+        const translations = version.VersionInfo.VersionTranslationInfo.Language.map(langauge => this.props.languages.map[langauge.languageCode].name);
```
Comment (sunpietro) [inline src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.js]: To shorten the line length, please create a variable: `const versionLanguages = version.VersionInfo.VersionTranslationInfo.Language;`

File: src/modules/universal-discovery/components/create/create.choose.language.component.js [hunk 2]
```diff
@@ -11,31 +11,32 @@ export default class ChooseLanguageComponent extends Component {
         this.renderOption = this.renderOption.bind(this);
 
         this.state = {
-            selectedLanguage: props.languages[0]
+            selectedLanguage: props.languages.map[props.languages.priority[0]]
```
Comment (sunpietro) [inline src/modules/universal-discovery/components/create/create.choose.language.component.js]: `.map` it's very confusing for JS developers.

**Review by sunpietro — CHANGES_REQUESTED** [review]

File: src/modules/universal-discovery/components/create/create.choose.language.component.js (hunk 2 above)
Comment (Nattfarinn) [reply src/modules/universal-discovery/components/create/create.choose.language.component.js]: What about `props.languages.thisIsNotAFunctionMap[props.languages.priority[0]]`? :P

Comment (sunpietro) [reply src/modules/universal-discovery/components/create/create.choose.language.component.js]: eqeqeq

**Review by mnocon — APPROVED** [review]
Looks good!

**Pull request merged**